docs/gadgets/index.mdx

```diff
@@ -6,11 +6,349 @@ description: Documentation about the official Gadgets"
+
+[Image-based](../reference/run.mdx#filtering-in-user-space):
+
+```
+A filter can match any field using the following syntax:
+  field==value     - matches, if the content of field equals exactly value
+  field!=value     - matches, if the content of field does not equal exactly value
+  field>=value     - matches, if the content of field is greater than or equal to the value
+  field>value      - matches, if the content of field is greater than the value
+  field<=value     - matches, if the content of field is less than or equal to the value
+  field<value      - matches, if the content of field is less than the value
+  field~value      - matches, if the content of field matches the regular expression 'value'
+               see [https://github.com/google/re2/wiki/Syntax] for more information on the syntax
+Multiple filters can be combined using a comma: field1==value1,field2==value2
+```
+
+Built-in:
+
+```bash
+A filter can match any column using the following syntax
+  columnName:value       - matches, if the content of columnName equals exactly value
+  columnName:!value      - matches, if the content of columnName does not equal exactly value
+  columnName:>=value     - matches, if the content of columnName is greater than or equal to the value
+  columnName:>value      - matches, if the content of columnName is greater than the value
+  columnName:<=value     - matches, if the content of columnName is less than or equal to the value
+  columnName:<value      - matches, if the content of columnName is less than the value
+  columnName:~value      - matches, if the content of columnName matches the regular expression 'value'
+                            see [https://github.com/google/re2/wiki/Syntax] for more information on the syntax
+This flag can be specified multiple times to combine multiple filters e.g. -F column1:value1 -F column2:value2
+```
+
+```bash
+# built-in
+$ sudo ig trace open -F 'path:/tmp/foo'
+
+# image-based
+# In this case the field was also renamed
+$ sudo ig run trace_open -F 'fname==/tmp/foo'
+```
+
+#### Showing or hidding columns
+
+In the built-in gadgets, the `-o columns=foo,bar` syntax was available for
+selecting which columns are shown. In the image-based gadgets this is done with
+the [--fields](../reference/run.mdx#selecting-specific-fields) flag:
+
+```bash
+# built-in
+$ sudo ig trace open -o columns=comm,path
+COMM             PATH
+...
+cat              /lib/libm.so.6
+cat              /lib/libresolv.so.2
+cat              /lib/libc.so.6
+cat              /tmp/foo
+
+# image-based
+$ sudo ig run trace_open --fields proc.comm,fname
+COMM             FNAME
+...
+cat              /lib/libm.so.6
+cat              /lib/libresolv.so.2
+cat              /lib/libc.so.6
+cat              /tmp/foo
+```
+
+#### Pulling Gadget Images
+
+Image-based Gadgets aren't shipped with the `ig` binary, they are instead stored
+as OCI artifacts on container registries. It means they need to be pulled from
```

Comment:
We sometimes say "container registries", sometimes "OCI registries". Which one is better?
```
$ git grep 'OCI registr'
$ git grep 'container registr'
```

Reply:
I think OCI registry is better.